### PR TITLE
Quality updates for `DockerExec` and some fixes and additions to Readme

### LINKED ISSUE #10: Installing on MacOS
If you install docker-proxy-stack on MacOS, you need to install a library that ships `realpath`.

Use brew: `brew install coreutils`

### LINKED ISSUE #11: Update the description for init certificates
The description says: Run `DockerExec create-cert` and follow the prompts.
The `DockerExec help` says, use `DockerExec do init-certs` for generating the initial certificates.

### PR DESCRIPTION
**Pull request by Sengorius**

### CODE REVIEW

**Review by sanklamm — APPROVED** [review]

**Review by sanklamm — APPROVED** [review]

File: readme.md [hunk 1]
```diff
@@ -73,6 +73,12 @@ certificates in the `certs` folder, containing a `rootCA.crt`. Any info you type
 into the prompts is optional. In the next step, you have to register this self-signed
 certificate to your default browser.
 
+#### Using the Proxy-Stack on MacOS
+
+The library `realpath` is mandatory for `DockerExec` to work with local files. You have
+to install the `coreutils` to get that library. With brew installed, use 
+`brew install coreutils`.
```
Comment (sanklamm) [inline readme.md:80]: add link to brew installation [Homepage of Homebrew](https://brew.sh/)

File: readme.md [hunk 2]
```diff
@@ -95,6 +101,9 @@ Use `docker network create --attachable "{YOUR_NETWORK_NAME}"` once to create
 the network. Don't change it afterwards. You can now test if the network was created with  
 `docker network ls`.
 
+Don't forget to add `127.0.0.1 docker.test` to your `/etc/hosts` (`sudo` necessary) file,
+which docker will use to redirect the requests on your local machine.
+
```
Comment (sanklamm) [inline readme.md:108]: Isn't this unnecessary now?

**Review by sanklamm — COMMENTED** [review]
added two comments

File: readme.md (hunk 1 above)
Comment (Sengorius) [reply readme.md:80]: Thanks! Was added to readme.

**Review by sanklamm — APPROVED** [review]

File: readme.md (hunk 2 above)
Comment (Sengorius) [reply readme.md:108]: I don't think so. If you do not wish to use the `DockerExec`, you machine still needs to know the domains of your dev network. So we better keep the annotation.

**Pull request merged**